src/js/components/Control.js

```diff
@@ -72,6 +72,13 @@ export default class Control {
     this.hide();
   }
 
+  _handleOnSelect = (value) => {
```

Comment:
I don't see the usage of this function anywhere. Is this function necessary ?

Reply:
I will delete it because it is not necessary